docs/install.md

```diff
@@ -14,8 +14,7 @@ store your docker registry details in a an environment variable. This
 install guide uses gcr on gke. 
 
 ```
-export GCP_PROJECT=<PROJECT_ID>
```

Comment:
What was the reason behind removing this?

Reply:
We only need the one global, not two now with these changes.